packages/stark-ui/assets/themes/_app-header-theme.scss

```diff
@@ -1,165 +1,143 @@
-/* TODO: remove after the app-header component is finished */
 .stark-app-header {
-  background: #123b71 url(/assets/images/app-header.png) top right no-repeat;
+  height: $stark-header-size;
```

Comment:
I think this file should be renamed since it contains more than a "theme" as it includes layouts, media queries, etc..